src/components/CommentList/CommentList.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+import Comment from '../Comment/Comment';
+
+export const CommentList = ({ comments }) => {
+  return (
+    <ul>
+      {comments.map(comment => 
+        <li>
```

Comment:
```suggestion
        <li key={comment.id}>
```
use `key` for `li`, not for the `Comment` component